blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -137,6 +141,8 @@ Deprecations
 Changes
 =======
 
+- Upgrade to Lucene 8.0.0
```

Comment:
Can we elaborate on the user facing consequences (scoring change?), are there any others?

Reply:
Not that I am aware of. We should decide on the codec format what will be supported in terms of backward compatibility.